### ISSUE TITLE
Identify and correct invalid collection temporal extents

### ISSUE THREAD

**Issue by anayeaye** [NASA-IMPACT/veda-data-pipelines#241]
# What
Some collection temporal extents are reported with timezone offsets rather than UTC. These extents need to be updated to reflect the temporal extent of the collection in UTC. This issue was uncovered here https://github.com/NASA-IMPACT/veda-data-pipelines/issues/228#issuecomment-1317426027 and was caused by a manual update of collection extents.

These are pretty easy to find by hand
```
SELECT id, collections."content"->'extent'->'temporal'->'interval'->0
FROM collections;
```

# Possibly out of scope
- Correcting or replacing the underlying PgSTAC methods used to update extents--that is covered here: https://github.com/NASA-IMPACT/veda-stac-ingestor/issues/25

# AC
- [ ] All collection extents are recorded as UTC

**Comment by anayeaye** [comment]
This issue is blocked by #270.

**Comment by j08lue** [comment]
No longer blocked, I guess.
